Doc/library/logging.rst

```diff
@@ -1151,37 +1151,40 @@ functions.
    |              | using the specified filename, rather than a |
    |              | StreamHandler.                              |
    +--------------+---------------------------------------------+
-   | ``filemode`` | Specifies the mode to open the file, if     |
-   |              | filename is specified (if filemode is       |
-   |              | unspecified, it defaults to 'a').           |
+   | ``filemode`` | If ``filename`` is specified, open the file |
```

Comment:
Use `*filename*` rather than ``` ``filename`` ``` (here and elsewhere).

From [the Documenting Python page in the developers guide](https://devguide.python.org/documenting/#id4):
> Names of local variables, such as function/method arguments, are an exception, they should be marked simply with \*var\*.